src/ControlSurface/Controls/Colorpicker/ScoreColorPoint.qml

```diff
@@ -2,40 +2,42 @@ import QtQuick 2.12
 import QtQuick.Controls 2.12
 
 Rectangle {
-    property string colorPointName: "ColorPointName"
-    property string colorPointPath: "path"
-    property int colorPointId
-    property string colorPointColor: "Color"
-    property string colorPointOpacity: "Opacity"
-    property string colorPointUuid
     id: colorButton
-    width: colorBackground.width / 20
     height: colorButton.width
     color: "#a7dd0d"
     border.width: 5
     border.color: "#a7dd0d"
+
+    property string controlCustom
+    property string controlDomain
+    property string controlValueType
+    property string controlValueData
+    property int controlId
+    property string controlUuid
+    property string controlSurfacePath
+
+    property color controlColor
     property color displayedColor: "#a7dd0dFF"
 
     Text {
         id: colorName
         anchors.left: colorButton.right
         anchors.leftMargin: 5
         anchors.horizontalCenter: colorButton.horizontalCenter
-        text: colorPointName
+        text: controlCustom
+        font.pointSize: background.height * ( 9 / 40)
         color: "#a7dd0d"
     }
     PanelBorder {
-        width: 100
-        height: 15
+        width: background.width * ( 2 / 3)
```

Comment:
:+1: the fewer hardcoded things the better it will scale on various devices & tablets